workers/data_refinery_workers/processors/create_compendia.py

```diff
@@ -189,11 +198,13 @@ def _perform_imputation(job_context: Dict) -> Dict:
     combined_matrix_zero = row_col_filtered_combined_matrix_samples
 
     # Transpose combined_matrix; transposed_matrix
-    transposed_matrix = combined_matrix_zero.transpose() #  row_col_filtered_combined_matrix_samples.transpose()
+    transposed_matrix_with_zeros = combined_matrix_zero.transpose() #  row_col_filtered_combined_matrix_samples.transpose()
```

Comment:
There was a question about why we transpose the matrix. We originally thought we were going to use KNN imputation and IIRC that expected standardize features, hence the transpose.